### DISCUSSION TITLE
installation difficulty

### DISCUSSION THREAD

**Discussion by dontfeedthegreen** [MRCIEU/TwoSampleMR#255] [Q&A]
Hi,
I've been trying to install this package with R 3.3.3 and I keep getting this error message:

*** arch - i386
Warning in library(pkg_name, lib.loc = lib, character.only = TRUE, logical.return = TRUE) :
  there is no package called 'TwoSampleMR'
Error: loading failed
Execution halted
*** arch - x64
Warning in library(pkg_name, lib.loc = lib, character.only = TRUE, logical.return = TRUE) :
  there is no package called 'TwoSampleMR'
Error: loading failed
Execution halted
ERROR: loading failed for 'i386', 'x64'

Not sure what I'm doing wrong? Advice welcome.
Thanks,

**Comment by mightyphil2000** [comment]
Please install the R package following these instructions:

https://mrcieu.github.io/TwoSampleMR/#installation

Sent from my iPhone

On 28 Aug 2017, at 15:02, dontfeedthegreen <notifications@github.com<mailto:notifications@github.com>> wrote:


Hi,
I've been trying to install this package with R 3.3.3 and I keep getting this error message:

*** arch - i386
Warning in library(pkg_name, lib.loc = lib, character.only = TRUE, logical.return = TRUE) :
there is no package called 'TwoSampleMR'
Error: loading failed
Execution halted
*** arch - x64
Warning in library(pkg_name, lib.loc = lib, character.only = TRUE, logical.return = TRUE) :
there is no package called 'TwoSampleMR'
Error: loading failed
Execution halted
ERROR: loading failed for 'i386', 'x64'

Not sure what I'm doing wrong? Advice welcome.
Thanks,

**Comment by dontfeedthegreen** [comment]
Thanks for your response. Those are the instructions I used, and that is the error message I got, after entering the install_github("MRCIEU/TwoSampleMR") command.

**Comment by mightyphil2000** [comment]
Do you have the package "devtools" installed?

Sent from my iPhone

On 29 Aug 2017, at 10:25, dontfeedthegreen <notifications@github.com<mailto:notifications@github.com>> wrote:


Thanks for your response. Those are the instructions I used, and that is the error message I got, after entering the install_github("MRCIEU/TwoSampleMR") command.

**Comment by dontfeedthegreen** [comment]
Yes, I have devtools installed.
